presto-base-jdbc/src/test/java/io/prestosql/plugin/jdbc/TestCachingJdbcClient.java

```diff
@@ -44,7 +47,7 @@
     private JdbcClient jdbcClient;
     private String schema;
 
-    @BeforeMethod
+    @BeforeClass
```

Comment:
`cachingJdbcClient` could not be shared between tests. Please extract it to @BeforeMethod.

Reply:
Why not?

Reply:
Ok, I see, It can, but I think it's does not worth to reuse stateful object under test between different methods. wdyt?